sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java

```diff
@@ -475,7 +475,7 @@ private List<UnresolvedExpression> singleFieldRelevanceArguments(
     // to skip environment resolving and function signature resolving
     ImmutableList.Builder<UnresolvedExpression> builder = ImmutableList.builder();
     builder.add(new UnresolvedArgument("field",
-        new Literal(StringUtils.unquoteText(ctx.field.getText()), DataType.STRING)));
```

Comment:
Just to confirm my understanding: string literal argument is unquoted and converted to qualified name here. So Analyzer will take care of it and throw semantic exception as needed? Any changes needed for the `multiFieldRelevanceArguments` below?

Reply:
@dai-chen that is correct.  So we're just piggy-backing on the existing checks. 
For multi fields, they should also be piggy-packed if they unquoted.

Reply:
I see. For single field, I think the current implementation is much cleaner and less intrusive. Thank you!